package.json

```diff
@@ -87,31 +35,117 @@
         "type": "string"
+      },
+      "order": 4
+    },
+    "global": {
+      "type": "object",
+      "order": 5,
+      "title": "Global ESLint",
+      "properties": {
+        "useGlobalEslint": {
+          "title": "Use global ESLint installation",
+          "description": "Make sure you have it in your $PATH",
+          "type": "boolean",
+          "default": false,
+          "order": 1
+        },
+        "globalNodePath": {
+          "title": "Global Node Installation Path",
+          "description": "Write the value of `npm get prefix` here",
+          "type": "string",
+          "default": "",
+          "order": 2
+        }
       }
     },
-    "rulesToDisableWhileFixing": {
-      "title": "Disable specific rules from fixes",
-      "description": "Prevent rules from being auto-fixed by ESLint. Applies to fixes made during saves as well as when running the `Linter Eslint: Fix File` command.",
-      "type": "array",
-      "default": [],
-      "items": {
-        "type": "string"
+    "disabling": {
+      "type": "object",
+      "order": 6,
+      "properties": {
+        "disableWhenNoEslintConfig": {
+          "title": "Disable when no ESLint config is found (in package.json or .eslintrc)",
+          "type": "boolean",
+          "default": true,
+          "order": 1
+        },
+        "ignoreFixableRulesWhileTyping": {
+          "title": "Ignore fixable rules while typing",
+          "description": "Have the linter ignore all fixable rules during linting when editing a document. The list is automatically updated on each lint job, and requires at least one run to be populated. Only supported when using ESLint v4+.",
+          "type": "boolean",
+          "default": false,
+          "order": 2
+        },
+        "rulesToDisableWhileFixing": {
+          "title": "Disable specific rules from fixes",
+          "description": "Prevent rules from being auto-fixed by ESLint. Applies to fixes made during saves as well as when running the `Linter Eslint: Fix File` command.",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string"
+          },
+          "order": 3
+        },
+        "rulesToSilenceWhileTyping": {
+          "title": "Silence specific rules while typing",
+          "description": "Useful when Atom fixes errors on save like `no-trailing-spaces` or `eol-last`.",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string"
+          },
+          "order": 4
+        }
       }
     },
-    "ignoreFixableRulesWhileTyping": {
-      "title": "Ignore fixable rules while typing",
-      "description": "Have the linter ignore all fixable rules during linting when editing a document. The list is automatically updated on each lint job, and requires at least one run to be populated. Only supported when using ESLint v4+.",
-      "type": "boolean",
-      "default": false
+    "advanced": {
```

Comment:
We can add a `collapsed` attribute to make this section collapsed by default.

Although it doesn't seem to be _documented_, it's [tested in the specs](https://github.com/atom/settings-view/blob/7689e9e04ae5b1f040a071611d372570ed518c50/spec/settings-panel-spec.coffee#L205).

Reply:
Filed https://github.com/atom/atom/issues/16030 about getting it officially documented 😉.